### DISCUSSION TITLE
获取动态数据异常

### DISCUSSION THREAD

**Discussion by abuuukk** [shanmiteko/LotteryAutoScript#380] [Q&A] [answered]
RT，一直频繁获取动态数据异常，
![QQ图片20240422233324](https://github.com/shanmiteko/LotteryAutoScript/assets/167792757/cb86f761-642a-4883-ad70-29a77487616d)
手动打开动态专栏是正常的，运行起来会异常

**Comment by shanmiteko** [comment, marked as answer]
#373
